README.md

```diff
@@ -1,10 +1,6 @@
-# eslint-config
+# Infinum JS Linters
 
-This repository contains two packages that provide Infinum's ESLint rules as [shareable configs](https://eslint.org/docs/developer-guide/shareable-configs.html):
-
-- [Base ESLint config](eslint-config/README.md)
-
-- [React ESLint config](eslint-config-react/README.md)
+ESLint and TSLint extensible shareable configs based on Infinum JavaScript Team's [coding styles and guidelines](https://infinum.com/handbook/books/frontend).
```

Comment:
Should we remove the mention of tslint?

Also, it might be useful to show the diagram here - we can draw a nicer one

Reply:
Here is a new one
![image](https://user-images.githubusercontent.com/480290/82536522-e0d18700-9b48-11ea-8fe1-b9f79c8c14b9.png)

Reply:
> Should we remove the mention of tslint?

I left in the TSLint Angular config, since that one we aren't yet migrating to ESLint.

> Also, it might be useful to show the diagram here

Nice, I'll add it.